sampleapp/src/main/scala/de/up/hpi/informationsystems/sampleapp/dactors/StoreSection.scala

```diff
@@ -80,17 +76,16 @@ object StoreSection {
 
       case GetVariableDiscountUpdateInventory.Request(customerId, cartTime, orderItems) =>
         getVariableDiscountUpdateInventory(customerId, cartTime, orderItems) match {
-          case Success(something) => sender() ! GetVariableDiscountUpdateInventory.Success(something)
+          case Success(totals: Seq[Record]) => sender() ! GetVariableDiscountUpdateInventory.Success(Relation(totals))
           case Failure(e) => sender() ! GetVariableDiscountUpdateInventory.Failure(e)
         }
     }
 
-    def getPrice(inventoryIds: Seq[Int]): Try[Seq[Record]] = {
+    def getPrice(inventoryIds: Seq[Int]): Try[Relation] = {
```

Comment:
We do not need a `Try` wrapping a `Relation` as it already contains a `Try`:
`Relation` --> `Try[Seq[Map[UntypedColumnDef, Any]]]`

Reply:
Thanks for catching these!